linetools/isgm/tests/test_init_emsys.py

```diff
@@ -0,0 +1,37 @@
+# Module to run tests on generating AbsSystem
+
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+# TEST_UNICODE_LITERALS
+
+import numpy as np
+import os
+
+import pytest
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+from linetools.isgm.emsystem import EmSystem
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'files')
+    return os.path.join(data_dir, filename)
+
+
```

Comment:
Add some other, simpler tests for instantiating the object.

Reply:
I've added a test for creating an EmSystem object